content/departments/cloud/working-agreements.md

```diff
@@ -1,8 +1,84 @@
-# Working agreements
+# Cloud Team Working Agreements
 
-Guidelines for how the [Cloud team](./index.md) collaborates. Also see [How we work](./index.md#how-we-work).
+Guidelines for how the [Cloud team](./index.md) collaborates.
 
-## Definition of Ready
+## Roles of Cloud teammates
+
+- [Cloud Engineer](#cloud-engineer)
+- [Cloud EM and PM](#cloud-em-and-pm)
+- [DRI](#dri)
+- [On-call Engineer](#on-call-engineer)
+- [Security Engineer](#security-engineer)
+
+## Issue tracking
+
+The [Cloud team GitHub Project](https://github.com/orgs/sourcegraph/projects/264/views/1) is the single source of truth.
+
+Learn more about [how we use GitHub Projects](./github-projects-beta.md)
```

Comment:
Based on the agreements done during the offsite we will need to update this section and the grooming and estimation sections as weel. 

I recommend we do it in separate PR